source/custom/nixnet_service.custom.cpp

```diff
@@ -0,0 +1,4 @@
+#include <nixnet/nixnet_service.h>
```

Comment:
FYI: You actually don't need to include this and it can make a big difference in compile time for bigger drivers (probably doesn't matter for xnet).